core/client/src/main/java/alluxio/client/block/RetryHandlingBlockWorkerClient.java

```diff
@@ -349,12 +349,9 @@ public void sessionHeartbeat(RetryPolicy retryPolicy) throws IOException, Interr
         Metrics.BLOCK_WORKER_HEATBEATS.inc();
         return;
       } catch (AlluxioTException e) {
-        AlluxioException ae = AlluxioException.fromThrift(e);
-        LOG.warn(ae.getMessage());
-        throw new IOException(ae);
-      } catch (ThriftIOException e) {
-        LOG.warn(e.getMessage());
-        throw new IOException(e);
+        AlluxioStatusException se = AlluxioStatusException.fromThrift(e);
+        LOG.warn(se.getMessage());
```

Comment:
Actually, I think we shouldn't log at all since we throw the exception. I've removed the warn